qa/tasks/mgr/test_module_selftest.py

```diff
@@ -194,7 +194,10 @@ def test_module_commands(self):
         self.wait_for_health(
             "Module 'selftest' has failed: Synthetic exception in serve",
             timeout=30)
-
+        # prune the crash reports, so that the health report is back to
+        # clean
+        self.mgr_cluster.mon_manager.raw_cluster_cmd(
+            "crash", "prune", "0")
```

Comment:
@tchaikov do you know why we started seeing this issue recently?

Reply:
it is a regression introduced by #41937. and because i thought that ignorelist would do the trick. see https://github.com/ceph/ceph/pull/41937#issuecomment-868731517, hence i merged it without retesting the changeset with 3edc04a46bff8912a2c9b623ab2ef6e05dbf7fba. my bad.

Reply:
Looks like this issue is not deterministic, probably that's why the TestTimeoutError (https://tracker.ceph.com/issues/51743) did not show up in your runs.

It showed up in https://pulpito.ceph.com/yuriw-2021-07-16_18:39:18-rados-wip-yuri-testing-master-7.16.21-distro-basic-smithi/, but not in the rerun https://pulpito.ceph.com/yuriw-2021-07-17_14:59:42-rados-wip-yuri-testing-master-7.16.21-distro-basic-smithi/.

It didn't show up in 10 runs of the test https://pulpito.ceph.com/nojha-2021-07-22_19:56:40-rados:mgr-wip-39871-distro-basic-smithi/

In any case, I don't see any issues with pruning the crashing reports, @liewegas WDYT?